spec/cypress/integration/Cookies.feature

```diff
@@ -0,0 +1,51 @@
+Feature: Cookie page
+  All ECF users needs to be able to set and modify their cookie preferences
+
+  Scenario: Preferences should not be initially set
+    Given I am on "cookie" page
+    Then "cookie consent" radios are unchecked
+
+  Scenario: Setting cookie preferences on cookie page
+    Given I am on "cookie" page
+
+    When I set "cookie consent" radio to "on"
+    And I click the submit button
+    Then cookie preferences have changed
+    And "cookie consent" radio with value "on" is checked
+
+    When I set "cookie consent" radio to "off"
+    And I click the submit button
+    Then cookie preferences have changed
+    And "cookie consent" radio with value "off" is checked
+
+  Scenario: Setting preferences through banner without js
+    Given I am on "index" page without JavaScript
+    When I click to accept cookies
+    Then cookie preferences have changed
+    And "cookie consent" radio with value "on" is checked
+
+  Scenario: Accepting cookies through banner with js
+    Given I am on "index" page
```

Comment:
There are many index pages - which one this refers to?

Reply:
Perhaps changing the name to Home Page would be better

Reply:
It's the /cookies as this file is cookies.feature